pkg/components/managementconsole/component.go

```diff
@@ -22,6 +22,49 @@ var ManagementConsoleComponent = Component{
 			{
 				IncludeAll: []string{"bz-Management Console"},
 			},
+			{Suite: "Administration pages pesudo translation"},
+			{Suite: "Branding check"},
+			{Suite: "Debug console for pods"},
+			{Suite: "Dynamic Plugins notification features"},
+			{Suite: "Dynamic plugins features"},
+			{Suite: "Dynamic provisioning"},
+			{Suite: "Home related pages via admin console"},
+			{Suite: "Notification drawer tests"},
+			{Suite: "Operators Installed page test"},
+			{Suite: "PDB List Page and Detail Page Test"},
+			{Suite: "Projects dropdown tests"},
+			{Suite: "about cluster setting page"},
+			{Suite: "add idp from console"},
+			{Suite: "admin console api related"},
+			{Suite: "alerts browser"},
+			{Suite: "command line tools page"},
+			{Suite: "console configs features"},
+			{Suite: "console feature on sno cluster"},
+			{Suite: "console style related"},
+			{Suite: "console-operator related"},
+			{Suite: "console-route"},
+			{Suite: "customize console related"},
+			{Suite: "dark-theme related feature"},
+			{Suite: "dashboards related cases"},
+			{Suite: "mega menu on console"},
+			{Suite: "namespace dropdown favorite test"},
+			{Suite: "operand form view"},
+			{Suite: "pod page"},
+			{Suite: "projects related features via web"},
+			{Suite: "query browser"},
```

Comment:
see from https://github.com/openshift-eng/ci-test-mapping/pull/85/files#r1607567823
`query browser` belongs to monitoring team